radixdlt/babylon-wallet-android
# Show arbitrary NFT metadata in NFT sheet

## Description
The NFT sheet is now able to display multiple NFT Metadata, besides the explicit ones. Primitive metadata are shown as strings and more complex ones like tuples, arrays, maps etc are previewed as "complex data"

## How to test
Load this profile and check the Empty2 account's only NFT.
[radix_wallet_backup_file.plaintext.json](https://github.com/radixdlt/babylon-wallet-android/files/13511883/radix_wallet_backup_file.plaintext.json)
```
remind index lift gun sleep inner double leopard exist sugar item whisper coast duty leopard law radar neutral odor tape finger position capital track
```

## Video
[NFT-metadata.webm](https://github.com/radixdlt/babylon-wallet-android/assets/125959264/352c35ca-e89c-4205-80a7-26cf4ecb3713)


## PR submission checklist
